examples/force_control/Force_control_example.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Inverse kinematics with the Kuka robot where the goal is to follow a moving sphere.
```

Comment:
Please update the description of the code to fit the functionalities coded here. Please be as descriptive as possible.